index.html

```diff
@@ -316,20 +316,20 @@ <h2>contact me</h2>
                 </p>
             </div>
             <div class="contact-me-form">
-                <form class="form" action="https://formspree.io/f/mvoyejav" id="form" method="post">
+                <form class="form" action="https://formspree.io/f/mvoyejav" method="post" onsubmit="return validateEmail()">
 
```

Comment:
- [ ] The form action is always called even if the validation fails. You would move it into your js script in order to trigger the submission only if validation pass. Let's fix it

Reply:
@EricMbouwe i really to get it your point or undertand

Reply:
i really don't get your point

Reply:
@EricMbouwe